<?xml version="1.0" ?><root>
<report><version>2</version><sessionid>df7e24e8eb4e7efa5ead4ee3</sessionid><type>desync</type><sku>ea.maxis.sims4.13.pc</sku><createtime>2020-05-02 20:37:21</createtime><buildsignature>Local.Unknown.Unknown.1.53.115.1020-1.200.000.400.Release</buildsignature><categoryid>__init__.py:457</categoryid><desyncid>df7e24e8eb4e7efa5ead4ee3</desyncid><systemconfig/><screenshot/><desyncdata>Liability &lt;interactions.utils.interaction_liabilities.CancelInteractionsOnExitLiability object at 0x08D7D430&gt; threw exception type object 'ComponentContainer' has no attribute 'set_last_user_directed_action_time' (AttributeError: type object 'ComponentContainer' has no attribute 'set_last_user_directed_action_time')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Core\sims4\utils.py", line 179, in wrapper&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\areaserver.py", line 206, in c_api_server_tick&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\zone.py", line 463, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\time_service.py", line 112, in update&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 217, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 385, in _resume&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 4107, in do_transition_single&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 4063, in _create_transition_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 3956, in run_super_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 3926, in run_direct_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 3811, in enter_si_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\si_state.py", line 935, in process_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\si_state.py", line 323, in remove_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\si_state.py", line 1134, in _remove_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4876, in on_removed_from_si_state&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4888, in _exited_pipeline&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 6653, in _exited_pipeline&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 4750, in release_liabilities&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\interaction_liabilities.py", line 309, in release&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4708, in cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4475, in _cancel_eventually&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\social\social_super_interaction.py", line 1545, in _cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4640, in _cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4814, in _update_autonomy_timer_on_cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 6710, in _update_autonomy_timer&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\__init__.py", line 457, in exported_func&#13;&#10;AttributeError: type object 'ComponentContainer' has no attribute 'set_last_user_directed_action_time'&#13;&#10;rtim=0</desyncdata></report>
<report><version>2</version><sessionid>df7e24e8eb4e7efa5ead4ee3</sessionid><type>desync</type><sku>ea.maxis.sims4.13.pc</sku><createtime>2020-05-02 20:37:24</createtime><buildsignature>Local.Unknown.Unknown.1.53.115.1020-1.200.000.400.Release</buildsignature><categoryid>__init__.py:457</categoryid><desyncid>df7e24e8eb4e7efa5ead4ee3</desyncid><systemconfig/><screenshot/><desyncdata>Exception in Sim Timeline: Exception resuming Element (AttributeError: type object 'ComponentContainer' has no attribute 'set_last_user_directed_action_time')&#13;&#10;Traceback (most recent call last):&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\scheduling.py", line 217, in simulate&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\elements.py", line 385, in _resume&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 4107, in do_transition_single&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 4063, in _create_transition_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\postures\transition_sequence.py", line 3956, in run_super_interaction&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 3926, in run_direct_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 3811, in enter_si_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\si_state.py", line 935, in process_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\si_state.py", line 323, in remove_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\si_state.py", line 1134, in _remove_gen&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4876, in on_removed_from_si_state&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4888, in _exited_pipeline&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 6653, in _exited_pipeline&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 4757, in release_liabilities&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 4750, in release_liabilities&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\utils\interaction_liabilities.py", line 309, in release&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4708, in cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4475, in _cancel_eventually&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\social\social_super_interaction.py", line 1545, in _cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4640, in _cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\super_interaction.py", line 4814, in _update_autonomy_timer_on_cancel&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\interactions\base\interaction.py", line 6710, in _update_autonomy_timer&#13;&#10;  File "T:\InGame\Gameplay\Scripts\Server\objects\components\__init__.py", line 457, in exported_func&#13;&#10;AttributeError: type object 'ComponentContainer' has no attribute 'set_last_user_directed_action_time'&#13;&#10;rtim=0</desyncdata></report>
</root>